inst/pages/exercises.qmd

```diff
@@ -1546,12 +1546,28 @@ tse <- transformAssay(tse, assay.type = "counts", method = "relabundance")
 #| code-fold: true
 #| code-summary: "Show solution"
 #| eval: false
+
 setSechmOption("hmcols", value=c("#F0F0FF","#007562"))
 sechm(
     tse, features = rownames(tse), assayName="relabundance",
     show_colnames=TRUE)
 ```
 
+5. Create a basic heatmap using the `pheatmap` package.
+
+```{r}
+#| label: heatmap-ex5
+#| code-fold: true
+#| code-summary: "Show solution"
+#| eval: false
+
+library(mia)
+library(pheatmap)
+
+# Create heatmap using pheatmap on mia's enterotype dataset
+pheatmap(assay(enterotype, "counts"))
```

Comment:
This is exercise 4 in Heatmap exercises.

In earlier heatmap exercises, "GlobalPatterns" data was used. However, here you use "esophagus". It would be simpler to use same dataset. (at least, the dataset should be loaded in this heatmap section)

Reply:
As this exercise 5 does the same as exercise 4, it should be say this. I think it is little bit confusing why we want to do the same.

-->

It could be justified by saying that we can create the same plot by using the ComplexHeatmap directly